lib/game_generator.rb

```diff
@@ -3,15 +3,18 @@
 require_relative './game_messenger'
 require_relative './game_message_generator'
 require_relative './game_state'
-require_relative './move_validator'
-require_relative './player'
+require_relative './input_validator'
+require_relative './human_player'
+require_relative './computer_player'
 require_relative './board'
 
 class GameGenerator
-  def create_a_game(user_interface)
+  def create_a_game(user_interface:, opponent: :human)
```

Comment:
Why default opponent is set to human? Ia it even possible to get through the get_opponent_selection loop without selecting an opponent? Do you need default here?